### DISCUSSION TITLE
[small detailing] [Calendar] make calendar hover partially light up surroundings

### DISCUSSION THREAD

**Discussion by CorruptionVR** [DustinBrett/daedalOS#411] [Feature Requests]
hey, I don't know if this is possible, but it would be a nice touch if the calendar hover would be more like the actual calendar hover
![imaage](https://github.com/DustinBrett/daedalOS/assets/78228404/f9bc09a6-b9b3-4ced-b330-6f92dab78f67)
Windows 10
![imaage (1)](https://github.com/DustinBrett/daedalOS/assets/78228404/27721e49-6a90-4a49-ba8c-e398c5717c7e)
daedalOS
this would also pave the way for a recreation of the settings app and a tiled start menu, as this is how it works in both of them.

**Comment by DustinBrett** [comment]
Thanks for the suggestion! That is indeed a good point and I am aware that my current styling code is focused on the hovering element and not it's surrounding elements.

I would like to improve that effect in general as it's also used in the start menu. I need to spend some time to think on an optimal way in React/Styled Components to have that effect appear on all surrounding elements.

![image](https://github.com/DustinBrett/daedalOS/assets/16656318/e9688b12-5f5c-43f3-8623-ecf4f1e48eb8)

**Comment by DustinBrett** [comment]
Here is a pretty decent example of doing it, but I recall my attempt to add it didn't go ideally in some cases.

https://codepen.io/gubb/pen/PdZqKy

**Comment by DustinBrett** [comment]
I finally did this. Took a while and I didn't use that CodePen, just thought about it and slowly made it work.

https://github.com/DustinBrett/daedalOS/commit/110ae274a444284bc7148d639df664d664341833, https://github.com/DustinBrett/daedalOS/commit/1e6a848ade2ac09786da9ac23b29090478509032, https://github.com/DustinBrett/daedalOS/commit/325047b68128c12384caced516ced7d2b83666c8

![image](https://github.com/DustinBrett/daedalOS/assets/16656318/3efba669-0a42-4651-a743-641d46d05088)

![image](https://github.com/DustinBrett/daedalOS/assets/16656318/b36a1507-3544-45e9-87d8-0f72b317f6a3)

![image](https://github.com/DustinBrett/daedalOS/assets/16656318/aa039d86-b328-4b44-8be4-d68d86e55057)
